### PR TITLE
docs(ingress): Gloo Edge Ingress Demo

### PR DESCRIPTION
**Pull request by nshankar13**
Create demo for using the Gloo Edge Ingress Controller with OSM

Signed-off-by: nshankar13 <nshankar@microsoft.com>

**Description**:

**Affected area**:

- New Functionality      [ ]
- Documentation          [X]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [ ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests                  [ ]
- CI System              [ ]
- Demo                   [ ]
- Performance            [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution? No

### CODE REVIEW

File: docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    ```yaml
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    metadata:
+    name: httpbin-ingress
+    namespace: httpbin
+    annotations:
+        kubernetes.io/ingress.class: gloo
+    spec:
+    rules:
```
Comment (allenlsy) [inline docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md:74]: Indentation issue here?

File: docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    - host: httpbin.httpbin.svc.cluster.local
+        http:
```
Comment (allenlsy) [inline docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md:76]: ditto

File: docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+1. Configure the `Upstream` object to use OSM's root ca bundle: 
+
+    ```yaml
+    apiVersion: gloo.solo.io/v1
+    kind: Upstream
+    metadata:
+    name: httpbin-httpbin-14001
+    namespace: gloo-system
+    spec:
+    sslConfig:
```
Comment (allenlsy) [inline docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md:102]: ditto

File: docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        namespace: gloo-system
+    kube:
+        selector:
+        app: httpbin
+        serviceName: httpbin
+        serviceNamespace: httpbin
+        servicePort: 14001
+    ```
+
+1. Confirm that a request to the httpbin service from the external IP address of the Ingress resource succeeds (in this case, the external address would be `52.234.160.38`)
```
Comment (allenlsy) [inline docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md:115]: Suggest change it to `a request to the the httpbin service via the external IP address of ...`.

My first impression was send request from ingress to httpbin

File: docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md (hunk 1 above)
Comment (nshankar13) [reply docs/content/docs/tasks_usage/traffic_management/demos/gloo_edge_ingress_demo.md:74]: Good catch, yes there are a few indentation issues in the yamls. Will correct these.

**Review by allenlsy — APPROVED** [review]

**Review by shashankram — APPROVED** [review]

**Review by draychev — APPROVED** [review]
Great! Excited to try this!

**Pull request merged**